docs/stac-api-spec/extensions/layer/README.md

```diff
@@ -0,0 +1,141 @@
+| `GET /layers/{layerID}` | [`GeoJSON Feature`](https://github.com/azavea/stac4s/blob/4aba9c6691fde1d5235f165454ceaef6c6b3e165/docs/stac-spec/extensions/layer/json-schema/layer-schema.json)                                                                                       | [STAC Layer](../../../stac-spec/extensions/layer/README.md) as a GeoJSON Feature. |
+
+## Schema
+
+- [JSON Static Layer Schema](../../../stac-spec/extensions/layer/json-schema/layer-schema.json)
+
+## How It Works
+
+When this extension is implemented, the API supports `layers` endpoint that list all layers available. An example of the result output can be found [here](../../../stac-spec/extensions/layer/examples/landsat-stac-layers/layers/catalog.json).
+
+The specific STAC Layer is available at `/layers/{layerID}`. The example output can be found [here](../../../stac-spec/extensions/layer/examples/landsat-stac-layers/layers/us.json)
+
+## Layer ID
+
+Layer ID is a unique identifier for a layer.
+
+## Example
+
+Request to `GET /layers`:
+
+```javascript
+{
+    "stac_version": "1.0.0-beta.1",
+    "stac_extensions": [],
+    "id": "layer-us-global",
+    "title": "Landsat 8 L1",
+    "description": "US STAC Layers, a STAC Catalog that represents a list of STAC Layers",
+    "links": [
+        {
+            "href": "/",
+            "rel": "root"
+        },
+        {
+            "href": "/layers",
+            "rel": "self"
+        },
+        {
+            "href": "/layers/layer-pa",
+            "rel": "item"
+        },
+        {
+            "href": "/layers/layer-us",
+            "rel": "item"
+        }
+    ]
+}
+```
+
+Request to `GET /layers/layer-us`:
+
+```javascript
+{
+    "type": "Feature",
+    "id": "layer-us",
+    "stac_extensions": [
+        "layer"
+    ],
+    // ... 
+    // bbox and geometry that cover all items that belong to the layer
+    "bbox": [
+        -101.40824987104652,
+        37.79718802132125,
+        -73.94863288954222,
+        41.41061537114088
+    ],
+    "geometry": {
+        "type": "Polygon",
+        "coordinates": [
+            [
+                [
+                    -101.40824987104652,
+                    37.79718802132125
+                ],
+                [
+                    -101.40824987104652,
+                    41.41061537114088
+                ],
+                [
+                    -73.94863288954222,
+                    41.41061537114088
+                ],
+                [
+                    -73.94863288954222,
+                    37.79718802132125
+                ],
+                [
+                    -101.40824987104652,
+                    37.79718802132125
+                ]
+            ]
+        ]
+    },
+    // properties can optionally contain layer temporal information
+    "properties": {
+        "start_datetime": "2018-05-01T00:00:00Z",
```

Comment:
These must have a `datetime` field -- but it's allowed to be `null` 🤦‍♂️. "required" is so confusing.

https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#properties-object

Reply:
@jisantuc is can be:

1. `properties = null` 
2. `datetime = null | ISO string`
3. if datetime = null than start_datetime and end_datetime should be set

Reply:
But `datetime` isn't `null` here -- it's missing. Even if it's null, it still has to be present: https://github.com/radiantearth/stac-spec/blob/dev/item-spec/json-schema/datetime.json#L48-L64

Reply:
I'm so baffled. What does `required` even mean if there are two branches, both say that `datetime` is a required field, and items without `datetime` still validate?

Reply:
@jisantuc I think json schema is not absolutely correct; `oneOf` actually means that validator would try to find the first best matching condition; 

However, if datetime were a mondatory field - how all items without datetime work?

Reply:
@jisantuc items follow the same schema https://github.com/radiantearth/stac-spec/blob/dev/item-spec/json-schema/item.json#L19 so I did not invent anything here.

Reply:
So as I already suggest - look into this URI https://www.jsonschemavalidator.net/s/C50aksbY

It is an interactive validator

Reply:
I think maybe that's not actually following schema links -- when I copy and paste the datetime schema on the left-hand side, the objects `{}` and `{"start_datetime": "2020-01-01T00:00:00Z", "end_datetime": "2020-01-01T00:00:00Z"}` don't validate and specifically complain about `datetime` being absent: https://www.jsonschemavalidator.net/s/nK74xAuQ

Reply:
@jisantuc oh I found a typo!

Reply:
@jisantuc I pushed changes! thanks for looking into that, I had a typo in the schema so it never checked datetimes. 

Yes, datetime should be described the same way it is described for items. Do you think it is bad? I would better follow the STAC Spec rather than implementing another new schema for datetime fields tbh.

Reply:
I'm confused, with the correct link, is it still saying items without a `datetime` key in their properties are valid?

Reply:
@jisantuc no, it works as you described originally now.

Reply:
Ah ok :tada: